src/SFA.DAS.PSRService.Web/Views/Report/Question/ReportNumbers.cshtml

```diff
@@ -1,4 +1,7 @@
 ﻿@model SFA.DAS.PSRService.Domain.Entities.Section
+@{
```

Comment:
This is new one - is this a Child => Child => Child view? - that could cause other problems!

Reply:
👍